power_grid_model_c/power_grid_model/include/power_grid_model/enum.hpp

```diff
@@ -72,6 +72,16 @@ enum class YBusElementType : IntS {
     fill_in_tf = 0b110
 };
 
+enum class FaultType : IntS {
+    three_phase = 0,
+    two_phase = 1,
+    two_phase_to_ground = 2,
+    single_phase_to_ground = 3,
+    default_value = na_IntS
```

Comment:
this is required if we want to add this to `update_data`

Reply:
True. The name of default value for enum might however be a bit confusing for user. I cant think of a better name though.

Reply:
`unknown`? `unspecified`?

Reply:
Equally vague. I guess we can leave it at default value for now.